### PR TITLE
Add Assign Department Functionality to App

### PR DESCRIPTION
**Pull request by wujy28**
# Add Assign Department Functionality to App
Add AssignCommand and AssignCommandParser classes to support the `assign` function the app intended to have.
---
# Key changes

## Implemented the functionality for assigning patients to departments
1. Add AssignCommand class to handle the execution of the command
2. Add AssignCommandParse class to parse user input
3. Integrate the command into AddressBookParser

Note: Testing has not yet been added.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -10,15 +10,15 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 
+import java.util.logging.Logger;
```
Comment (longnguyentan) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:13]: Good effort in using Logger!

File: src/main/java/seedu/address/logic/commands/AssignCommand.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import seedu.address.model.patient.IcNumber;
+import seedu.address.model.patient.Name;
+import seedu.address.model.patient.Patient;
+import seedu.address.model.patient.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Assigns a patient to a hospital department
+ */
+public class AssignCommand extends Command {
```
Comment (longnguyentan) [inline src/main/java/seedu/address/logic/commands/AssignCommand.java:30]: Good design of `AssignCommand`

**Review by longnguyentan — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (thienmy0) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:13]: agree!

**Review by thienmy0 — APPROVED** [review]
LGTM!

**Pull request merged**